integrationtests/pom.xml

```diff
@@ -32,6 +32,18 @@
             <version>1.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.kroxylicious.testing</groupId>
+            <artifactId>testing-junit5-extension</artifactId>
+            <version>0.1</version>
```

Comment:
extract the version number as a maven property

Reply:
Done, thanks!

Reply:
I'm not entirely sure why we do both things. I'd be happy embedding versions in the dependency management... thoughts @k-wall